buildlib/tools/coverity.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -eExl
+reldir=$(dirname $0)
+
+source ${reldir}/common.sh
+#
+# Run Coverity and report errors
+# The argument is a UCX build type: devel or release
+#
+run_coverity() {
+	echo 1..1 > coverity.tap
+
+	modules_for_coverity
+
+    if [ $res -gt 0 ]; then
+    	azure_log_issue "All modules cannot load"
+    fi
+
+	ucx_build_type=$1
+
+	xpmem_root=$(module show dev/xpmem-latest 2>&1 | awk '/CPATH/ {print $3}' | sed -e 's,/include,,')
```

Comment:
can't just load xpmem module?
if ucx can't recognize xpmem after module is loaded - pls fix xpmem check in ucx and/or xpmem module to export more env vars